content/03.results.md

```diff
@@ -61,6 +61,14 @@ Here, vg clearly outperformed other approaches (Figures {@fig:2}a and {@fig:hgsv
 In non-repeat regions and insertions across the whole genome, the F1 scores and precision-recall AUC were higher for vg compared to other methods.
 For example, for deletions in non-repeat regions, the F1 score for vg was 0.824 while the second best method, Paragraph, had a F1 score of 0.717.
 We observed similar results when evaluating the presence of an SV call instead of the exact genotype (Figures {@fig:2}a and {@fig:hgsvc-real}).
+
```

Comment:
I added this paragraph to the results, to mention the "over-genotyping" problem in some of the other methods. What do you think?